<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
     <item>
      <widget class="QTabWidget" name="tabWidget_settings">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="tab_units">
        <attribute name="title">
         <string>units</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1">
           <item>
            <widget class="QLabel" name="label_angleType">
             <property name="text">
              <string>angle display type</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_angleType"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1">
           <item>
            <widget class="QLabel" name="label_distanceType">
             <property name="text">
              <string>distance display type</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_distanceType"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,1">
           <item>
            <widget class="QLabel" name="label_temperatureType">
             <property name="text">
              <string>temperature display type</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_temperatureType"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,1">
           <item>
            <widget class="QLabel" name="label_angleDigits">
             <property name="text">
              <string>angle display digits</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_angleDigits"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,1">
           <item>
            <widget class="QLabel" name="label_distanceDigits">
             <property name="text">
              <string>distance display digits</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_distanceDigits"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,1">
           <item>
            <widget class="QLabel" name="label_temperatureDigits">
             <property name="text">
              <string>temperature display digits</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_temperatureDigits"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_columnSettings">
        <attribute name="title">
         <string>column settings</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QTabWidget" name="tabWidget_columnSettings">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="tab_featureAttributes">
            <attribute name="title">
             <string>display feature attributes</string>
            </attribute>
           </widget>
           <widget class="QWidget" name="tab_TrafoParamAttributes">
            <attribute name="title">
             <string>display trafoParam attributes</string>
            </attribute>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_plugins">
        <attribute name="title">
         <string>plugins</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="1,2">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QTreeView" name="treeView_plugins">
               <property name="alternatingRowColors">
                <bool>false</bool>
               </property>
               <property name="autoExpandDelay">
                <number>-1</number>
               </property>
               <property name="animated">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_6" stretch="1,2">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>description:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QTextBrowser" name="textBrowser"/>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,6">
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>author:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEdit_3"/>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="1,6">
                 <item>
                  <widget class="QLabel" name="label_3">
                   <property name="text">
                    <string>version:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEdit_2"/>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="1,6">
                 <item>
                  <widget class="QLabel" name="label_4">
                   <property name="text">
                    <string>file path:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEdit"/>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QWidget" name="widget_pluginInfo" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_cancel">
         <property name="text">
          <string>cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_ok">
         <property name="text">
          <string>ok</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
